CHANGES.md

```diff
@@ -5,10 +5,132 @@ all changes see the [closed
 milestones](https://github.com/cylc/cylc/milestones?state=closed) for each
 release.
 
+-------------------------------------------------------------------------------
+## __cylc-7.8.0 (2018-11-27)__
+
+Minor release with over 120 issues closed. Significant issues include:
+
+### Enhancements
+
+[#2809](https://github.com/cylc/cylc/pull/2809) - __auto-migration__ - suite
+server programs can now be told (via global config) to self-migrate to
+another available host, e.g. for server maintenance. (The pool of cylc hosts 
+should see a shared filestem).
+
+[#2614](https://github.com/cylc/cylc/pull/2614) and 
+[#2821](https://github.com/cylc/cylc/pull/2821) - __web-based job log viewer__ -
+ `cylc review` (migration of "Rose Bush" from the Rose project). 
+
+[#2693](https://github.com/cylc/cylc/pull/2693) - __auto host selection__ -
```

Comment:
You may want to bundle this with #2809 above.

Reply:
Done.